var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -17,9 +17,11 @@ class Hdf5VolLog(AutotoolsPackage):
 
     version("master-1.1", branch="master")
 
-    version("1.1.0", commit="ca146fa7d320ec5c0b397669b330c78fceeabb57")
```

Comment:
Old versions are usually deprecated rather than deleted as they *may* exist in someone's cache.  See https://spack.readthedocs.io/en/latest/packaging_guide.html?highlight=deprecate#deprecating-old-versions.

Reply:
In this case, version 1.1.0 is still available but was updated from a commit to the tag logvol.1.1.0 tag in the vol-log-based source repo.  The hdf5 dependency minimum version was updated to the currently available 1.13.2 version from 1.13.0 which is no longer available.